spring-grpc-dependencies/pom.xml

```diff
@@ -112,6 +113,16 @@
 				<artifactId>proto-google-common-protos</artifactId>
 				<version>${google-common-protos.version}</version>
 			</dependency>
+			<dependency>
```

Comment:
Please replace this w/ `micrometer-bom` which handles all `io.micrometer` version constraints.